pkgs/by-name/dl/dleyna/package.nix

```
@@ -0,0 +1,62 @@
+{
+  stdenv,
+  lib,
+  docutils,
+  fetchFromGitLab,
+  meson,
+  ninja,
+  pkg-config,
+  gupnp_1_6,
+  gupnp-av,
+  gupnp-dlna,
+  sphinx,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "dleyna";
+  version = "0.8.3";
+
+  outputs = [
+    "out"
+    "dev"
+  ];
+
+  setupHook = ./setup-hook.sh;
+
+  src = fetchFromGitLab {
+    domain = "gitlab.gnome.org";
+    owner = "World";
+    repo = "dLeyna";
+    tag = "v${finalAttrs.version}";
+    sha256 = "sha256-ti4yF8sALpWyrdQTt/jVrMKQ4PLhakEi620fJNMxT0c=";
+  };
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    docutils
+    sphinx
+    pkg-config
+  ];
+
+  buildInputs = [
+    gupnp_1_6
+    gupnp-av
+    gupnp-dlna
```

We prefer listing direct dependencies explicitly:

https://gitlab.gnome.org/World/dLeyna/-/blob/d20a8f008ad39f1d0bfe274295deececebeb24de/meson.build#L87-95

```suggestion
    gupnp-dlna
    gssdp_1_6
    libsoup_3
    libxml2
```